src/_cost-storage.ts

```
@@ -0,0 +1,204 @@
+ * @returns File path for the session's cost data
+ */
+function getCostFilePath(sessionId: string): string {
+	return join(getCostStorageDir(), `${sessionId}.json`);
+}
+
+/**
+ * Ensure the cost storage directory exists
+ * @returns Result indicating success or failure
+ */
+function ensureCostStorageDir(): Result<void, Error> {
+	return Result.try({
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)